Bottom: 8747e136893cbae47a628f7ac36c798d5e24a15e
Top:    0fd7c247c5040cdd82ac0ed3c10320b7f52da119
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-08-27 15:44:21 +0100

Refresh of try-to-make-gdk-and-gl-version

---

diff --git a/src/hid/gtk/gtkhid-gdk.c b/src/hid/gtk/gtkhid-gdk.c
index d4e1746..36f4f39 100644
--- a/src/hid/gtk/gtkhid-gdk.c
+++ b/src/hid/gtk/gtkhid-gdk.c
@@ -924,14 +924,8 @@ ghid_drawing_area_new (GHidPort *port)
   return gtk_drawing_area_new ();
 }
 
-gboolean
-ghid_start_drawing (GHidPort *port)
-{
-  return TRUE;
-}
-
 void
-ghid_end_drawing (GHidPort *port)
+ghid_pinout_preview_init (GhidPinoutPreview *preview)
 {
 }
 
@@ -958,6 +952,17 @@ ghid_drawing_area_configure_hook (GHidPort *port)
     }
 }
 
+gboolean
+ghid_start_drawing (GHidPort *port)
+{
+  return TRUE;
+}
+
+void
+ghid_end_drawing (GHidPort *port)
+{
+}
+
 void
 ghid_screen_update (void)
 {
@@ -984,11 +989,6 @@ ghid_drawing_area_expose_cb (GtkWidget *widget,
   return FALSE;
 }
 
-void
-ghid_pinout_preview_init (GhidPinoutPreview *preview)
-{
-}
-
 gboolean
 ghid_pinout_preview_expose (GtkWidget *widget,
                             GdkEventExpose *ev)
diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index c6ab190..02c592b 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -818,6 +818,11 @@ ghid_pinout_preview_init (GhidPinoutPreview *preview)
                                 GDK_GL_RGBA_TYPE);
 }
 
+void
+ghid_drawing_area_configure_hook (GHidPort *port)
+{
+}
+
 gboolean
 ghid_start_drawing (GHidPort *port)
 {
@@ -848,11 +853,6 @@ ghid_end_drawing (GHidPort *port)
 }
 
 void
-ghid_drawing_area_configure_hook (GHidPort *port)
-{
-}
-
-void
 ghid_screen_update (void)
 {
 }
diff --git a/src/hid/gtk/gui.h b/src/hid/gtk/gui.h
index 2487b2e..054f304 100644
--- a/src/hid/gtk/gui.h
+++ b/src/hid/gtk/gui.h
@@ -478,7 +478,7 @@ void ghid_logv (const char *fmt, va_list args);
 /* gui-pinout-window.c */
 void ghid_pinout_window_show (GHidPort * out, ElementTypePtr Element);
 
-/* gtkhid-gdk.c OR gtkhid-gl.c */
+/* gtkhid-gdk.c AND gtkhid-gl.c */
 hidGC ghid_make_gc (void);
 void ghid_destroy_gc (hidGC);
 void ghid_use_mask (int use_it);
@@ -500,13 +500,13 @@ void ghid_invalidate_all ();
 void ghid_show_crosshair (gboolean show);
 void ghid_init_renderer (int *, char ***, GHidPort *);
 GtkWidget *ghid_drawing_area_new (GHidPort *port);
+void ghid_pinout_preview_init (GhidPinoutPreview *preview);
+void ghid_drawing_area_configure_hook (GHidPort *port);
 gboolean ghid_start_drawing (GHidPort *port);
 void ghid_end_drawing (GHidPort *port);
-void ghid_drawing_area_configure_hook (GHidPort *port);
 void ghid_screen_update (void);
 gboolean ghid_drawing_area_expose_cb (GtkWidget *, GdkEventExpose *,
                                       GHidPort *);
-void ghid_pinout_preview_init (GhidPinoutPreview *preview);
 gboolean ghid_pinout_preview_expose (GtkWidget * widget, GdkEventExpose * ev);
 GdkPixmap *ghid_render_pixmap (int cx, int cy, double zoom,
                                int width, int height, int depth);
